core-data/selectors.js

```diff
@@ -69,3 +69,14 @@ export function getMedia( state, id ) {
 export function getPostType( state, slug ) {
 	return state.postTypes[ slug ];
 }
+
+/**
+ * Return theme suports data in the index.
+ *
+ * @param {Object} state    Data state.
+ *
+ * @return {Mixed?}     Index data.
```

Comment:
- `Mixed` is not a valid type. JSDoc encourages `*` as the wildcard "any" type ([reference](http://usejsdoc.org/tags-param.html#multiple-types-and-repeatable-parameters))
- Extra whitespace should be removed after types, both in `@param` and `@return`

Reply:
Handled in 1c6e6936